mednafen/pce_fast/vdc.cpp

```diff
@@ -849,7 +843,11 @@ void VDC_RunFrame(EmulateSpecStruct *espec, bool IsHES)
          };
 
          DisplayRect->x = 0;
-         DisplayRect->w = ws[correct_aspect][vce.dot_clock];
+         if(hoverscan == 1 && vce.dot_clock == 1){
```

Comment:
They are the different display modes for what I understand, a bit like on the SNES modes if I'm not too wrong.
[The resolutions they use are those.](https://github.com/westonlast/beetle-pce-fast-libretro/pull/1/files/2a42aea68b748ed18977fb89ca728c137d1df9bf#diff-06f9c9d982271c1b652d735c0030471eR50)
vce.dot_clock 1 is that 341/352 mode which usually gets its overscan cut for whatever technical reason I don't know but I try to conform to.